OTAnalytics/domain/flow.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from OTAnalytics.domain.section import Section
+
+
+@dataclass(frozen=True)
+class FlowId:
+    id: str
+
+    def serialize(self) -> str:
+        return self.id
+
+
+class Flow:
+    def __init__(
+        self,
+        id: FlowId,
+        start: Section,
+        end: Section,
+        distance: float,
+    ) -> None:
+        self.id: FlowId = id
+        self.start: Section = start
+        self.end: Section = end
+        self._distance: float = distance
+
+    def distance(self) -> float:
```

Comment:
How can we overwrite the distance? Simply rebuild the flow for now?

Reply:
At this point yes, generate a new flow and save it to the repository. After OP#2123 is merged, the distance is editable.